cmd/command.go

```diff
@@ -250,6 +250,24 @@ func mGet(cli *Client, args [][]byte) (interface{}, error) {
 	return values, err
 }
 
+func getRange(cli *Client, args [][]byte) (interface{}, error) {
+	if len(args) != 3 {
+		return nil, newWrongNumOfArgsError("getrange")
+	}
+	key, start, end := args[0], args[1], args[2]
+	s, err := strconv.Atoi(string(start))
```

Comment:
Variable names should be more clear, such as `start`、`end`， otherwise others maybe do not know what it means.

Reply:
start, err := strconv.Atoi(string(arg[1]))

Reply:
I understand.
I will fix it.